upt_macports/upt_macports.py

```diff
@@ -18,10 +18,11 @@ def create_package(self, upt_pkg, output):
     def _render_makefile_template(self):
         env = jinja2.Environment(
             loader=jinja2.PackageLoader('upt_macports', 'templates'),
-            trim_blocks=True,
+            trim_blocks=False,
             lstrip_blocks=True,
             keep_trailing_newline=True,
         )
```

Comment:
what about we just use the defaults here (or remove these four lines)? The defaults are:
```
trim_blocks=False
lstrip_blocks=False
keep_trailing_newline=False
``` 
we definitely want the last one set to False as we don't want an empty line at the end of the Portfile